src/content/en/updates/2020/03/devtools.md

```diff
@@ -30,21 +30,19 @@ different types of vision deficiencies experience your site.
 DevTools can emulate blurred vision and the following
 [types of color vision deficiencies](http://www.colourblindawareness.org/colour-blindness/types-of-colour-blindness/):
 
-* Protanopia. The inability to perceive red light.
-* Protanomaly. A reduced sensitivity to red light.
-* Deuteranopia. The inability to perceive green light.
-* Deuteranomaly. A reduced sensitivity to green light.
-* Tritanopia. The inability to perceive blue light.
-* Tritanomaly. A reduced sensitivity to blue light (extremely rare).
-* Achromatopsia. The inability to perceive any color except for shades of grey (extremely rare).
-* Achromatomaly. A reduced sensitivity to green, red, and blue light (extremely rare).
-
-The `-anomaly` forms are the (rare) extreme versions of the `-ia` forms. Every person
-with an `-ia` vision deficiency is different and might see things differently (being able
-to perceive more/less of the relevant colors). The DevTools emulations are just a visual
-approximation of how someone might experience one of these vision deficiencies. Although the
-approximation should be good enough for you to identify and resolve issues, there's no way
-to simulate exactly what a given person would experience.
+* Protanopia: the inability to perceive red light.
```

Comment:
What do you think of a bit stronger phrasing here? Like "the complete inability to" or "the inability to perceive any"?

Reply:
Done